mailet/standard/src/test/java/org/apache/james/transport/mailets/LogMessageTest.java

```diff
@@ -86,11 +89,13 @@ public void serviceShouldFailWhenMailHasNoStream() throws Exception {
                 .build();
         mailet.init(mailetConfig);
 
+        MimeMessage message = new MimeMessage(Session.getDefaultInstance(new Properties()));
```

Comment:
what's wrong with the builder ?

Reply:
The wrapping phase generates an empty body, which do not trigger the bug generating that log message. 

I will add a commit for having that mailet handling well messages with no body.

Reply:
I made Log mailet more resilient by relying on the DataHandler, which do not trigger this tested bug anymore.